Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerForMainServer.java

```diff
@@ -19,16 +19,12 @@
  */
 public final class GameServerForMainServer extends UnicastRemoteObject implements IGameServerForMainServer {
 
-    private final GameServerRmiController controller;
-
     /**
      * Initializes the GameServerForMainServer class.
      *
-     * @param controller RmiController that is given, not null
      * @throws RemoteException
      */
-    public GameServerForMainServer(GameServerRmiController controller) throws RemoteException {
-        this.controller = controller;
+    public GameServerForMainServer() throws RemoteException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1186)